compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -3992,6 +4049,7 @@ TR::Register *OMR::X86::TreeEvaluator::arraycopyEvaluator(TR::Node *node, TR::Co
 
    TR::DataType dt = node->getArrayCopyElementType();
    uint32_t elementSize = 1;
+   static bool avoidREPMOVSW = feGetEnv("TR_AvoidREPMOVSW");
```

Comment:
Please provide a slightly more descriptive env var to indicate context.  e.g., `TR_AvoidREPMOVSWForArrayCopy`

Reply:
Sure, it has been updated.